### DISCUSSION TITLE
怎么用pcl2下载文件aaaa

### DISCUSSION THREAD

**Discussion by Atlanti5** [Meloong-Git/PCL#1058] [提问] [answered]
怎么用pcl2下载文件aaaa

**Comment by WForst-Breeze** [comment, marked as answer]
### 手  把  手  教  学（）
1. 右键目标文件的下载链接，复制下载链接（若您已有其他方式获取下载链接，请忽略这步）
![1674911722552](https://user-images.githubusercontent.com/110760354/215268332-8e61bf96-4c70-4f80-9c59-04b30974f8fa.png)

2. 进入 PCL2 **更多-百宝箱-下载自定义文件-下载地址** 中粘贴地址，并填写文件名
![image](https://user-images.githubusercontent.com/110760354/215268419-84ea39d4-e88f-4ba9-bad7-91165dd824f6.png)

3. 单击“开始下载”，等待下载完成

**Reply by Atlanti5** [reply]
谢谢<3
